### PR TITLE
New article "Curation and review process for the Edge Add-ons store"

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article for review:
https://review.learn.microsoft.com/microsoft-edge/extensions-chromium/publish/add-ons-store-curation?branch=pr-en-us-2868

AB#47048444

### CODE REVIEW

File: microsoft-edge/extensions-chromium/publish/add-ons-store-curation.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Edge Web Store Curation and Reviews
+description: How submissions to the Microsoft Edge Web Store are curated and reviewed by the store team.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/add-ons-store-curation.md:3]: @Bhuvana-Priya - check the added `description:`.

File: microsoft-edge/extensions-chromium/publish/add-ons-store-curation.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+<!-- ====================================================================== -->
+## Organizing Edge Add-Ons Store items
+
+The store organizes the items to help users discover the ones that are relevant to their interests.
+
+Users can also discover the items they are interested in on the Edge Add-Ons Store in several ways such as the home page, search bar, multiple curated collections, and categories.
+
+The following factors are used to organize items:
+* **Quality:** Items featured on the Edge Add-Ons Store are selected based on their eligibility criteria.
+* **Relevance:** Ranking takes into consideration factors such as the item name, description relevancy, popularity, and user experience.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/add-ons-store-curation.md:26]: @Bhuvana-Priya - .docx has "relevancy".  Either word seems ok; I might research more about the two words.
https://ahdictionary.com/word/search.html?q=relevancy

File: microsoft-edge/extensions-chromium/publish/add-ons-store-curation.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+---
+title: Edge Web Store curation and reviews
+description: How submissions to the Microsoft Edge Web Store are curated and reviewed by the store team.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 10/13/2023
+---
+# Edge Web Store curation and reviews
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/extensions-chromium/publish/add-ons-store-curation.md:10]:
```suggestion
# Curation and review process for submissions to the Edge Web Store
```
if change, will change in all the usual spots

**Review by Bhuvana-Priya — APPROVED** [review]

File: microsoft-edge/extensions-chromium/publish/add-ons-store-curation.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/extensions-chromium/publish/add-ons-store-curation.md:10]: official name is "Edge Add-Ons Store" - 32 hits in 10 articles in this Docs repo.  Revised Suggestion above.

**Pull request merged**